plone/app/iterate/browser/control.py

```diff
@@ -39,6 +40,15 @@ class Control(BrowserView):
     This is a public view, referenced in action condition expressions.
     """
 
+    def _can_lock(self):
+        try:
+            if lock.isLocked(self.context):
+                return False
```

Comment:
Is this correct? Does it return False if it is locked? It cannot be locked but it implements ILockable.